src/src/DataFixtures/AppFixtures.php

```diff
@@ -2,16 +2,44 @@
 
 namespace App\DataFixtures;
 
+use App\Entity\Question;
+use App\Factory\AnswerFactory;
+use App\Factory\QuestionFactory;
+use App\Factory\TagFactory;
```

Comment:
Not relative to the PR
```suggestion
```

Reply:
@MisterAzix

Reply:
Pas grave . t'façon j'aurai besoin de ca